client/verta/verta/client.py

```diff
@@ -48,6 +47,8 @@
 from .endpoint import Endpoints
 from .endpoint.update import DirectUpdateStrategy
 from .visibility import _visibility
+from ._protos.public.uac import WorkspaceV2_pb2 as _Workspace
```

Comment:
See my comment in `_organization.py`

```suggestion
from ._protos.public.uac import WorkspaceV2_pb2
```